docs/source/api/apollo-server.mdx

```diff
@@ -475,6 +475,21 @@ If this is set to any string value, use that value instead of the environment va
 </td>
 </tr>
 
+<tr>
+<td>
+
+###### `status400WithErrorsAndNoData`
+
+`boolean`
+
+</td>
+<td>
+
+This option is a v4 regression mitigation that we recommend users set to `true`. This option will be ignored in Apollo Server v5 and its `true` behavior will become the default. Visit the [migration guide](../migration/#known-regressions) for more details on the regression.
```

Comment:
```suggestion
**Set this option to `true`.** It mitigates a regression introduced in Apollo Server v4 where the server returns a 200 status code (instead of 400) when a client query provides invalid variables. [Learn more.](../migration/#known-regressions)

Apollo Server v5 will _always_ behave as if this option is `true` (and will ignore any provided value).
```